pkgs/applications/graphics/rendersvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pango,
+  gdk_pixbuf,
+  pkgconfig,
+  qtbase
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "rendersvg-${version}";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "RazrFalcon";
+    repo = "resvg";
+    rev = "v${version}";
+    sha256 = "1bnm00w6lqvk43g49j22as3l4frz99gpiv8rm22ys3d3ka56r5z2";
+  };
+
+  cargoSha256 = "1ahw4q6nyq6jqxy1lcpbwyf3ip38cwc7lcsk6z0kjdcr7ngf71nn";
+
+  nativeBuildInputs = [
+    pkgconfig
+    pango
+    gdk_pixbuf
+    qtbase
```

Comment:
Shouldn't the last three be in `buildInputs`?

Reply:
done :)